app/js/layout/App.jsx

```diff
@@ -2,11 +2,13 @@ import React, { Component, Fragment } from 'react';
 import { HashRouter, Route } from "react-router-dom";
 import { connect } from 'react-redux';
 import { Container } from 'reactstrap';
+import EmbarkJS from 'Embark/EmbarkJS'; // Needed for auto reload
```

Comment:
I trust you, could you just describe why?

Reply:
Here or in the comment in the code?

Reply:
I can see we are using EmbarkJS below so I get the import bu why the `// Needed for auto reload`

Reply:
Oh, because at the time of me coding, we weren't actually using EmbarkJS. I just rebased so I didn't see we are using it.
The reason why we need it for reload is because we add the Websocket snippet inside EmbarkJS and I haven't figured out how we can extract that snippet and put it inside the actual code.

Reply:
Understood do you think we should remove the comment?

Reply:
Yeah good idea